pkgs/by-name/kc/kcl/package.nix

```diff
@@ -58,18 +56,7 @@ buildGoModule rec {
   # By default, libs and bins are stripped. KCL will crash on darwin if they are.
   dontStrip = stdenv.hostPlatform.isDarwin;
 
-  # env vars https://github.com/kcl-lang/kcl-go/blob/main/pkg/env/env.go#L29
-  postFixup = ''
-    wrapProgram $out/bin/kcl \
-      --prefix PATH : "${
-        lib.makeBinPath [
-          kclvm
-          kclvm_cli
-        ]
-      }" \
-      --prefix KCL_LIB_HOME : "${lib.makeLibraryPath [ kclvm ]}" \
-      --prefix KCL_GO_DISABLE_INSTALL_ARTIFACT : false
-  '';
+  doCheck = !(with stdenv; isDarwin && isAarch64); # turning off tests in this case as ofBorg does not ternimate waiting for a builder
```

Comment:
```suggestion
  doCheck = !(with stdenv; isDarwin && isAarch64); # turning off tests in this case as ofBorg does not terminate waiting for a builder
```
*typo

Reply:
Done. Changed to:

`doCheck = !(with stdenv; isDarwin && isAarch64); # turning off tests in this case as ofBorg endlessly waits for a builder
`